integration-tests/features/components.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Components API V1
+
+  Scenario: Check the API entry point
+    Given System is running
+    When I access /api/v1/
+    Then I should get 200 status code
+
```

Comment:
@tisnik we will have to check the authentication module as well. Could you add steps for checking error 401 as well for /component-search and /component-analyses?

(Here is the location](https://github.com/fabric8-analytics/fabric8-analytics-server/tree/master/hack/auth_test_fixtures) of test auth token generation code. We can reuse this code to generate the Auth token for calling the API methods.